README.md

```
@@ -1,9 +1,20 @@
+
+Required commands
+-----------------
+
+* rebar3
+* beep
 
 Build
 -----
 
-    $ rebar3 compile
+    $ rebar3 escriptize
```

markdownlint-cli2 (0.17.2)

Dollar signs used before commands without showing output
null

(MD014, commands-show-output)